connect_ext_ppr/models/replicas.py

```diff
@@ -11,3 +12,30 @@ class Product(Model):
     logo = db.Column(db.String(2000), nullable=False)
     version = db.Column(db.SmallInteger())
     owner = db.Column(db.String(255))
+
+
+class ProductItem(Model):
```

Comment:
Well we can handle that based on new incoming product version: if that new version have a new item, we create new record, otherwise we can update existing item, because we need to have this data when someone use "regenerate" ppr's action. Or you suggest to do this?